### ISSUE TITLE
API Browser Broken after Upgrade to 2.3 (stored_fields)

### ISSUE THREAD

**Issue by gruselglatz** [Graylog2/graylog2-server#4042]
The API browser doesnt work anymore if you give the query the parameter "field". You will get this message:

```
Unable to perform search query.\n\nThe field [fields] is no longer supported, please use [stored_fields] to retrieve stored fields or _source filtering if the field is not stored"
```

## Expected Behavior
Use the new stored_fields

## Current Behavior
using fields

## Possible Solution
?

## Steps to Reproduce (for bugs)
1. got to ..../api-browser#!/Search/Relative/searchRelative_get_1
2. fill the form and set the "fields"
3. press "Try it out!"
4. ERROR

## Context
Using the API Browser for a lot of trial and error.

## Your Environment
Graylog 2.3.0+81f8228 on syslog.xyz.local (Oracle Corporation 1.8.0_141 on Linux 3.10.0-514.16.1.el7.x86_64)
api version: 2.3.0+81f8228

**Comment by joschi** [comment]
@gruselglatz Which version of Elasticsearch are you using?

**Comment by joschi** [comment]
This seems to be caused by the renaming of the `fields` field to `stored_fields` in Elasticsearch 5.x:

* https://www.elastic.co/guide/en/elasticsearch/reference/5.5/breaking_50_search_changes.html#_literal_fields_literal_parameter
* https://github.com/elastic/elasticsearch/pull/20166

**Comment by gruselglatz** [comment]
@joschi Yes, i am using ES 5.5

**Comment by tholu** [comment]
Possible related problem with `fields` (but using Elasticsearch 2): #4068

**Comment by joschi** [comment]
@tholu No, this is a different issue.

**Comment by joschi** [comment]
Fixed in #4069 and #4077.

@tholu I have to take that back. It *was* related. 😉
